### ISSUE TITLE
XamlUIBasics SemanticZoom Sample broken when built with newer Microsoft.NETCore.UniversalWindowsPlatform 

### ISSUE THREAD

**Issue by Noemata** [Microsoft/Windows-universal-samples#980]
The SemanticZoom.ZoomedOutView mode does not render correctly when building with newer versions of Microsoft.NETCore.UniversalWindowsPlatform. The rendering problem (blank content) is so severe it makes applications that leverage this feature unusable.

I also consider all of the Windows-universal-samples to be invalid at this point because most of the samples are being built against the 5.0 version of Microsoft.NETCore.UniversalWindowsPlatform. These samples should reveal issues with newer 6.x Platform SDK builds so that Microsoft can see the problems that have been introduced so they can be fixed before vaulting such problems onto the developer community and ultimately Windows 10 users.

There are many other issues related to newer versions of Microsoft.NETCore.UniversalWindowsPlatform, that work perfectly with the 5.0 version.

Many bugs have been introduced starting with the 6.x releases of Microsoft.NETCore.UniversalWindowsPlatform. One of the more egregious is the way ZipFile.ExtractToDirectory no longer honors the FutureAccessList. Please start building with the latest releases!

The reason some Microsoft staffers appear mystified when presented with issues by other developers on here is because the external developers have swapped in 6.x versions of Microsoft.NETCore.UniversalWindowsPlatform when building these samples. Hence they see issues Microsoft staffers do not.

**Comment by oldnewthing** [comment]
I think this is mostly a platform problem, not a samples problem. The only samples-related part is "Please have the samples use version 6.x instead of 5.0."

**Comment by Noemata** [comment]
I think it is only reasonable to assume that the samples here represent fully working solutions.  If the samples do not work on a particular "platform" variant then please don't publish them.  These samples are supposed represent core functionality that we as developers can count on.  The samples are excellent, by the way.  It's sad that the "platform problem" has become an integral part of why these samples have become invalid.  There is no point in publishing code samples that don't actually work in the field.  External developers do not possess the theoretical environment these samples appear to be targeting. Please either stop publishing code that is theoretically working, or have the platform folks directly involved in triaging the issues that pop up here.  The "platform problem" excuse is invalidating many of these samples.

**Comment by oldnewthing** [comment]
We can certainly look at adding workarounds in the samples for platform problems, but remember that the samples are reliant upon the platform. I think your specific issue here is that the Semantic Zoom feature doesn't work with version 6.x of the .NET Framework? Maybe that's why the sample uses version 5.0: Sticking with 5.0 is the workaround. (I don't know for sure, just wondering.) So the sample does work - it applies the workaround.

**Comment by Noemata** [comment]
I disagree with your platform assertion, particularly as it relates to SDK variations 5.x through 6.x, given these are supposed to be progressive improvements and enhancements on what was there before.  Building with a newer version of Microsoft.NETCore.UniversalWindowsPlatform should only make things better (add features, run faster, fix old bugs).  The fact that old capabilities are not preserved and bugs creep into existing capabilities is a serious issue.  What is the incentive for developers to progress if we can expect code rot to pervade newer releases.  Using 5.0 is not a workaround.  It's a giant step backwards.  I'm very disappointed that a seasoned individual such as @oldnewthing would propose such a thing.  Please initiate the steps needed to fix these issues, else adoption of UWP will continue to suffer.  The fact that these samples use 5.0 appears to act as a disincentive to address regression problems.

**Comment by oldnewthing** [comment]
I did not write the samples in this repo. I just try to keep the repo running. If your suggestion is that the samples should be updated to use 6.x and all the problems related to 6.x be fixed, then that's a great suggestion. But if you're saying "The samples don't work with 6.x, please provide a workaround", then the answer is "The samples use 5.0. That's the workaround."

**Comment by Noemata** [comment]
I'm ok with "samples should be updated to use 6.x and all the problems related to 6.x be fixed".  Thank you.
